src/screens/Menu/Menu/components/MenuFooter.js

```diff
@@ -27,11 +27,13 @@ import { useTranslationWithPrefix } from 'translations/translate';
 
 // Components
 import Button from 'components/core/Button';
-import Modal from 'components/Modal';
-import PrismicDocumentModal from 'components/Modals/PrismicDocumentModal';
 
 // Constants
-import { MENU_SYSTEM_INFORMATION, BACKUP_WALLET_IN_SETTINGS_FLOW } from 'constants/navigationConstants';
+import {
+  MENU_SYSTEM_INFORMATION,
+  BACKUP_WALLET_IN_SETTINGS_FLOW,
+  TERMS_AND_PRIVACY,
```

Comment:
Could we change this to LEGAL_MODAL or something similar for consistency with the rest of the naming please?